stylesheets/bootstrap.scss

```diff
@@ -72,4 +72,13 @@
 // Custom breadcrumb separator
 .breadcrumb > li + li::before {
     content: ">"
-}
\ No newline at end of file
+}
+
+pre {
```

Comment:
It's tested and working on my end; note that it doesn't exactly match the Stack Overflow post.

A random page that I made long enough to wrap:

![image](https://cloud.githubusercontent.com/assets/3310349/25561327/7ef1709a-2d1e-11e7-9128-0f4560149b7e.png)